### PR TITLE
New Package: pinentry

### PR DESCRIPTION
**Pull request by yuui-tanabe**
I created pinentry dependency package for gnupg 2

### CODE REVIEW

**Comment by yuui-tanabe** [comment]
The CI Passed. What should I do?

**Comment by Alexpux** [comment]
Fix my comments

**Comment by yuui-tanabe** [comment]
How do I fix your comments?

**Comment by Alexpux** [comment]
I leave comments in PKGBUILD. Fix issues and push here

**Comment by yuui-tanabe** [comment]
How can I find it ?

**Comment by Alexpux** [comment]
What find? Open tab files changed and see my comments

**Comment by mati865** [comment]
@Alexpux there are no comments on this PR.

**Comment by Alexpux** [comment]
See it now?

**Comment by yuui-tanabe** [comment]
OK, I'll create a new PR after fixed

File: mingw-w64-pinentry/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
+pkgbase=mingw-w64-pinentry
```
[the remark this answers is not in the document]
Comment (yuui-tanabe) [reply mingw-w64-pinentry/PKGBUILD]: Is it like Maintainer:* ?

[the remark this answers is not in the document]
Comment (Alexpux) [reply mingw-w64-pinentry/PKGBUILD]: Yes

**Pull request closed without merging**